packages/backend/src/errors.ts

```diff
@@ -37,3 +41,18 @@ export const scimErrorHandler = (
         next(error); // Pass the error to the next middleware if it's not a ScimError
     }
 };
+export const slackErrorHandler = (e: unknown, context: string) => {
```

Comment:
exported here because i need it in both SlackClient and Slackbot

Reply:
[non-blocker] can be a static method on `SlackError`

Reply:
will leave this here since it matches well with the other handlers 👍